delete.ps1

```diff
@@ -0,0 +1,251 @@
+##################################################
+# HelloID-Conn-Prov-Target-Eduarte-Medewerker-Delete
```

Comment:
De Delete is kopie van Disable. Alleen is de logging ook nog 1 op 1 gelijk. Ik weet niet wat hier normaal wordt toegepast, maar ik zou toch wel willen zien dat het om een delete gaat.

Reply:
De action is delete. Die wordt meegegeven in het audit bericht. (zie je niet in de code doet HelloID zelf). Verder is de message wat er daadwerkelijk gebeurt is. Ook de entitlements staan los voor elkaar.

Mogelijk wel een goed punt om mee te nemen, wat is een goede manier van Auditlogging als je tijdens een delete alleen een disable doet.

Voor nu doe ik er even niks mee.